packages/core/src/lib/unique-id.ts

```diff
@@ -0,0 +1,41 @@
+import { setContext, getContext } from 'svelte';
+import { readable, type Readable, get } from 'svelte/store';
+
+type GetCount = () => number;
+type UniqueIdStore = Readable<GetCount>;
+
+const CONTEXT_KEY = 'prime-core-uid';
+
+/**
+ * Initialize a store to  track ID counts to ensure uniqueness
+ *
+ * ```ts
+ * // +layout.svelte
+ * import { initializeUniqueIdStore } from '$lib/unique-id';
+ * initializeUniqueIdStore();
+ * ```
+ */
+export const initializeUniqueIdStore = () => {
```

Comment:
In the SvelteKit app, we've been naming things that provide a context with `provide`. So here:

```suggestion
export const provideUniqueID = () => {
```